docs/tutorial.md

```diff
@@ -481,32 +557,38 @@ conflicts.
 
 Let's try one final command for changing the contents of an exiting commit. That
 command is `jj diffedit`, which lets you edit the contents of a commit without
-checking it out.
+checking it out (i.e. the "right side").
 
 ```shell
 $ jj diffedit -r @-
-Using default editor ':builtin'; you can change this by setting ui.diff-editor
+Hint: Using default editor ':builtin'; run `jj config set --user ui.diff-editor :builtin` to disable this message.
 Created kwtuwqnm 70985eaa (empty) ABC
 Rebased 1 descendant commits
+Working copy now at: mrxqplyk 1c72cd50 (conflict) ABCD
+Parent commit      : kwtuwqnm 70985eaa (empty) ABC
+Added 0 files, modified 1 files, removed 0 files
+There are unresolved conflicts at these paths:
+file    2-sided conflict
 New conflicts appeared in these commits:
   mrxqplyk 1c72cd50 (conflict) ABCD
 To resolve the conflicts, start by updating to it:
   jj new mrxqplykmyqv
 Then use `jj resolve`, or edit the conflict markers in the file directly.
 Once the conflicts are resolved, you may want to inspect the result with `jj diff`.
 Then run `jj squash` to move the resolution into the conflicted commit.
-Working copy now at: mrxqplyk 1c72cd50 (conflict) ABCD
-Parent commit      : kwtuwqnm 70985eaa (empty) ABC
-Added 0 files, modified 1 files, removed 0 files
 ```
 
-In the diff editor, edit the right side by e.g. adding something to the first
-line. Press 'c' to save the changes and close it. You can now inspect the rewritten
-commit with `jj diff -r @-` again and you should see your addition to the first
-line. Unlike `jj squash -i`, which left the content state of the commit
-unchanged, `jj diffedit` (typically) results in a different state, which means
+In this case the default diff editor is not powerful enough to allow you to
+actually do much other than picking hunks from the existing diff.
+Using an alternative diff editor as described above you can change the commit
+entirely and add or delete content as you wish.
+You can now inspect the rewritten commit with `jj diff -r @-` again, and you
+should see your addition to the first line.
+Unlike `jj squash -i`, which left the content state of the commit unchanged,
+`jj diffedit` (typically) results in a different state, which means
 that descendant commits may have conflicts.
```

Comment:
Something like this (changing only what's necessary from the original):

> In the diff editor, use the arrow keys and spacebar to select all lines but the last. Press 'c' to save the changes and close it. You can now inspect the rewritten
commit with `jj diff -r @-` again and you should see your deletion of the last
line. Unlike `jj squash -i`, which left the content state of the commit
unchanged, `jj diffedit` (typically) results in a different state, which means
that descendant commits may have conflicts.

Reply:
Done in https://github.com/martinvonz/jj/commit/b2692a325f251969f212153df4d02b26b772efb0